docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/add-certificate-self.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Add self-signed certificates to YugabyteDB Anywhere
+headerTitle: Add certificates
+linkTitle: Add certificates
+description: Add self-signed certificates to YugabyteDB Anywhere.
+headcontent: Use your own certificates for encryption in transit
+menu:
+  preview_yugabyte-platform:
+    parent: enable-encryption-in-transit
+    identifier: add-certificate-1-self
+    weight: 20
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../add-certificate-self/" text="Self-Signed" active="true" >}}
+{{<tabitem href="../add-certificate-ca/" text="CA-Signed" >}}
+{{<tabitem href="../add-certificate-hashicorp/" text="Hashicorp Vault" >}}
+{{<tabitem href="../add-certificate-kubernetes/" text="Kubernetes" >}}
+{{</tabs>}}
+
+Instead of using YugabyteDB Anywhere-provided certificates, you can use your own self-signed certificates that you upload to YugabyteDB Anywhere.
+
+## Prerequisites
+
+The certificates must meet the following criteria:
+
+- Be in the `.crt` format and the private key must be in the `.pem` format, with both of these artifacts available for upload.
+- Contain IP addresses of the target database nodes or DNS names as the Subject Alternative Names (wildcards are acceptable).
```

Comment:
```suggestion

```

In this case, they are giving us a CA key, so no node certificates are involved. There is some miscommunication in this whole page, will clarify below.